.lando.yml

```diff
@@ -0,0 +1,75 @@
+name: simplytest-me
+recipe: drupal9
+
+config:
+  php: "7.3"
+  webroot: web
+  xdebug: false
+  database: mariadb:10.3
+
+events:
+  # Clear cache after a database import
+  post-db-import:
+    - appserver: cd $LANDO_WEBROOT && drush cache-rebuild --yes
+
+services:
+  appserver:
+    run:
+      - cd $LANDO_MOUNT && composer install --no-scripts
+
+  nodejs:
+    type: node:14
+    globals:
+      gulp-cli: latest
+    run:
+      - cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm install
+      - cd $LANDO_MOUNT/web/profiles/contrib/simplytest/themes/simplytest_theme && npm run build
```

Comment:
Why is this on `run` and not `build`?

Reply:
I think this depends on the intended workflow. If I come back to this project in a few months and pull the latest updates, then I may find it convenient to have the theme build automatically.

I guess the same idea applies to `composer install`: are we trying to make this convenient for the active developer or the once-in-a-while developer?

Reply:
Great question @benjifisher. The project I used as a basis for this PR is geared toward quick onboarding of a new developer who has never installed the project before.